### DISCUSSION TITLE
Keep getting a Ruby gem 403

### DISCUSSION THREAD

**Discussion by danglingptr0x0** [lowlighter/metrics#926] [Q&A] [answered]
Hey,

When running the classic template, I keep getting the following error for a gem that the script is trying to install:
```ERROR:  While executing gem ... (Gem::RemoteFetcher::FetchError)
[2620](https://github.com/simonSlamka/simonSlamka/runs/5371666269?check_suite_focus=true#step:2:2620)
bad response Forbidden 403 (https://api.rubygems.org/quick/Marshal.4.8/nokogiri-1.13.3-x64-unknown.gemspec.rz)
```

It worked before and then suddenly stopped. I keep getting the 403 even through `wget`, so I know the issue is not here, but I wonder if it works for others and if so, where are your scripts getting that particular gem from ... ?

Thanks

**Comment by danglingptr0x0** [comment, marked as answer]
Oh my God, I'm so embarrassed ... I forgot to fetch from upstream ... XD

Sorry

**Reply by lowlighter** [reply]
No worries !
Yeah I vaguely remember this issue being related to ruby version, I think it was patched when the dockerfile was updated from buster to bullseye
